code/datums/map_config.dm

```diff
@@ -81,12 +93,16 @@
 
 	defaulted = FALSE
 
-#define CHECK_EXISTS(X) if(!istext(json[X])) { log_world(X + "missing from json!"); return; }
+#define CHECK_EXISTS(X) if(!istext(json[X])) { log_world(X + " missing from json!"); return; }

```

Comment:
could you change it to "[##X] missing from json!" please

Reply:
What does [## do? Is it some sort of fancy printing of the object?

Reply:
##X will be replaced by the argument X in a define. So if you called CHECK_EXISTS("butthole") and butthole didn't exist, it'd do ```log_world("["butthole"] missing from json!")```